src/lib/dlt_user.c

```diff
@@ -3769,6 +3862,23 @@ DltReturnValue dlt_user_log_send_log(DltContextData *log, int mtype)
 
 #   endif
 
+#ifdef DLT_TRACE_LOAD_CTRL_ENABLE
+            /* check trace load before output */
+            if (!sent_size)
+            {
+                if (!dlt_user_check_trace_load(log->log_level, time_stamp,
```

Comment:
In the body of dlt_user_check_trace_load, you are trying to check and send the warning message (via dlt_user_output_internal_msg).
So to me, there could be an infinity loop here!!??

Reply:
@ysk-sato Any idea here?